dashboard/db/migrate/20160805190505_drop_user_trophies_and_trophies.rb

```diff
@@ -0,0 +1,6 @@
+class DropUserTrophiesAndTrophies < ActiveRecord::Migration
+  def up
```

Comment:
Ah, my mistake!  I thought omitting `down` made it irreversible but I am wrong.  The way you have it written is how I would expect it to work (can be rolled back, but rolling back is a no-op).